mathics/builtin/tensors.py

```diff
@@ -535,9 +540,8 @@ class VectorQ(Builtin):
      = True
     """
 
+    summary_text = "test whether an object is a vector"
```

Comment:
I don't get this change. It was in sorted order (which makes the changes above clear that you haven't been following) and now it is not.